### PR TITLE
Removing crash issue

### PR DESCRIPTION
**Pull request by KevinPlus3**
Updated code to remove possibility of Python script crashing if a command line parameter does not have '=' in it.  Also, reduces the number of loops down to 1 instead of 2 - yes, number of lines of code is not as concise but alas.

BUT, before committing to merging, need to decide if we wish to go on with script if a parameter does not have '=' in it and assume all appropriate parameters are still inputted, OR do we wish to raise an error if the such a possibility arises.

### CODE REVIEW

File: MasterScripts/systemprep-linuxmaster.py [hunk 1]
```diff
@@ -325,9 +325,16 @@ def main(noreboot='false', **kwargs):
 
 
 if "__main__" == __name__:
-    #convert command line parameters of the form `param=value` to a dict
-    kwargs = dict(x.split('=', 1) for x in sys.argv[1:])
-    #Convert parameter keys to lowercase, parameter values are unmodified
-    kwargs = dict((k.lower(), v) for k, v in kwargs.items())
+    # Convert command line parameters of the form `param=value` to a dict
+    # If CL param does not contain '=', store param as a key with None value
+    # NOTE: Do we prefer this program to raise error if there is a CL param with no '=' in it?
```
Comment (lorengordon) [inline MasterScripts/systemprep-linuxmaster.py]: For now, raise an error. Longer term, would like to convert this to proper option parsing.

**Comment by lorengordon** [comment]
Looks good to me, ready to merge or still working?

**Comment by KevinPlus3** [comment]
Yes, ready to merge. The transition to argparse will be a bigger involvement and should be tracked in a new branch since it will rip this stuff out.  Also will transition to using log module for better messaging in this script in a different branch.

**Comment by lorengordon** [comment]
:ship:

**Pull request merged**